docs/guides/transfer_learning.md

```diff
@@ -0,0 +1,166 @@
+---
+jupytext:
```

Comment:
I'm confused, is the notebook generated from this file? But shouldn't it be in the build dir then?

Reply:
@cgarciae PTAL

Reply:
Re-posting here just in case:

Hey @jheek, using jupytext we are syncing the .ipynb and .md files, the .ipynb will be rendered but the .md are there to ease the review process.